packages/core/src/Cardano/util/numeric.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable no-use-before-define */
+
+export declare type UInt64 = number;
+export declare type UInt32 = number;
+export declare type Int64 = number;
```

Comment:
int64 > Number.MAX_SAFE_INTEGER
```suggestion
export declare type UInt64 = bigint;
export declare type UInt32 = number;
export declare type Int64 = bigint;
```

Reply:
addressed here: https://github.com/input-output-hk/cardano-js-sdk/pull/388/commits/f93bd1a2bac1896cc1289c422db283dd6dfd92dd

This implied a bigger refactor than I was expecting. Let me know if it was done correctly

Reply:
Yep, looking good :+1: